generation/WinSDK/enums.json

```diff
@@ -24494,11 +24494,12 @@
   },
   {
     "name": "LVTILEVIEWINFO_FLAGS",
-    "members": [
-      {
-        "name": "LVTVIF_EXTENDED"
-      }
-    ],
+    "flags": true,
+    "autoPopulate": {
+      "filter": "LVTVIM_|LVTVIF_",
```

Comment:
This needs an exclude. But I wasn't sure how to achieve that.

Reply:
`LVTVIF_EXTENDED`, I tried `(?!LVTVIF_EXTENDED)(LVTVIM_|LVTVIF_)` in the filter. I didn't spin my wheels too much on it.

Reply:
I ended up fixing it manually. But it would still be cool to be able to exclude a value.